io.appflowy.AppFlowy.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Name=AppFlowy
+Icon=io.appflowy.AppFlowy
+Exec=env GDK_GL=gles app_flowy
```

Comment:
this exec line is wrong. Like in the previous PR.

It should the name of the executable. Nothing else. If you need to set the env, then you should add a wrapper script

Reply:
Thanks. I have updated the code.